fdbserver/QuietDatabase.actor.cpp

```diff
@@ -48,17 +48,23 @@ ACTOR Future<WorkerInterface> getMasterWorker( Database cx, Reference<AsyncVar<S
 	loop {
 		state vector<WorkerDetails> workers = wait( getWorkers( dbInfo ) );
 
-		for( int i = 0; i < workers.size(); i++ ) {
-			if( workers[i].interf.address() == dbInfo->get().master.address() ) {
-				TraceEvent("GetMasterWorker").detail("Stage", "GotWorkers").detail("MasterId", dbInfo->get().master.id()).detail("WorkerId", workers[i].interf.id());
-				return workers[i].interf;
-			}
-		}
-
-		TraceEvent(SevWarn, "GetMasterWorkerError")
+		if(dbInfo->get().master.present()) {
```

Comment:
Is this supposed to be:

```suggestion
		if(!dbInfo->get().master.present()) {
```